requirePromise.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 module.exports = function requirePromise() {
-	if (typeof Promise !== 'function') {
+	if (typeof (Promise || global.Promise || window.Promise) !== 'function') {
```

Comment:
These are three identical ways of stating the same thing, except that it will break in browsers (which don't have `global`).